.pre-commit-config.yaml

```diff
@@ -21,12 +21,6 @@ repos:
       exclude: ^infra/scoped/task-definitions
     - id: check-yaml
 
-- repo: https://github.com/detailyang/pre-commit-shell.git
```

Comment:
This check is removed as it failed the pre-commit check and did not provide useful output (the script passes when sent through https://www.shellcheck.net/).

Reply:
fwiw the way this pre-commit tool works is it fails if it makes any changes, so a second attempt works. I had a bit of confusion with that - still think it's worth removing this specific check